### ISSUE TITLE
A question regarding the experimental results

### ISSUE THREAD

**Issue by max-andr** [TransEmbedBA/TREMBA#1]
Dear authors,

Thanks for the interesting paper. The idea to search for adversarial examples in a latent space of another network is quite interesting.

I have a question regarding the experimental results. In Table 6 the NES method for VGG-19 has 100% success rate, and needs 651 queries in average:
![image](https://user-images.githubusercontent.com/14852704/69221564-dfaea580-0b77-11ea-893a-b968bf4ceda8.png)

While in the [Bandits](https://arxiv.org/abs/1807.07978) paper they reported for a VGG-16-BN network (yes, it's not *exactly* the same as VGG-19, but still) that NES achieves **89.5%** success rate, and needs 764 queries in average.
![image](https://user-images.githubusercontent.com/14852704/69221748-4c29a480-0b78-11ea-8429-90aa32c2f960.png)

So you report a *significantly* better success rate (10.5% difference) with less queries in average. Do you have some ideas where this difference can come from?

Thank you.

**Comment by TransEmbedBA** [comment]
I think there are several differences.

1. We do not use PGD in the experiment while Bandits use it. 
2. We choose different sample size from Bandits. In our experiment, it seems large sample size will sightly increase the number of queries for NES.
3. When it comes to the success rate, we set the query limit to 50000 while Bandits set it to 10000. I think this makes a huge difference for the success rate.
4. We use C&W loss while Bandits use cross entropy.
5. We also involves momentum and learning rate decay in the experiment but we are not sure whether Bandits use them.

I am not sure the fraction of the contribution of the each component. But I think these differences may explain the difference between our experiment and Bandits.

**Comment by max-andr** [comment]
> We do not use PGD in the experiment while Bandits use it.

So you mean you rather use the trick with tanh() to make sure that the perturbation is in the Linf-box, thus avoiding any projections?
And also you further optimized the hyperparameters of NES, used the C&W loss, the momentum term, and the learning rate decay to get a stronger baseline to compare to?

**Comment by TransEmbedBA** [comment]
PGD means the sign function outside the gradient estimation. I do not use tanh() for the NES baseline.

**Comment by max-andr** [comment]
Ok, I see. Thanks for the clarification!
